modules/op-resolver-http.adoc

```diff
@@ -0,0 +1,100 @@
+// This module is included in the following assemblies:
+// * create/remote-pipelines-tasks-resolvers.adoc
+
+// // *openshift_pipelines/remote-pipelines-tasks-resolvers.adoc
+:_mod-docs-content-type: PROCEDURE
+[id="resolver-http-specify_{context}"]
+= Specifying a remote pipeline or task with the HTTP Resolver
+
+When creating a pipeline run or task run, you can specify a remote pipeline or task from an HTTP or HTTPS URL.
+
+.Prerequisites
+
+* You can optionally configure the service account for the HTTP resolver.
+
+.Procedure
+
+* To specify a remote pipeline or task from an HTTP URL, use the following reference format in the `pipelineRef` or `taskRef` spec:
++
+[source,yaml]
+----
+# ...
+  resolver: http
+  params:
+  - name: url
+    value: <fully_qualified_http_url>
+  - name: serviceAccount
```

Comment:
Same, there is no notion of serviceAccount here, see https://github.com/tektoncd/pipeline/blob/main/docs/http-resolver.md